frontend/src/components/UserGenPage.js

```diff
@@ -47,10 +46,11 @@ class UserGenPage extends Component {
     // Displays the existing one
     if (this.props.nickname != null) {
       this.setState({
-        nickname: this.props.nickname,
```

Comment:
I removed `nickname` form this component and directly access to the one on `AppState` otherwise managing all states when recovering a robot were really hard to manage, it has to be tested on the web frontend